src/components/SiteDetail.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
```

Comment:
I'm noticing that you're not using `PropTypes`. Why is that? Don't you love maintainable code and puppies?